transaction-exclusion-api/core/src/testFixtures/kotlin/net.consensys.linea.transactionexclusion.test/Common.kt

```diff
@@ -41,3 +41,26 @@ val defaultRejectedTransaction = RejectedTransaction(
     nonce = 100UL
   )
 )
+
+val rejectedContractDeploymentTransaction = RejectedTransaction(
+  txRejectionStage = RejectedTransaction.Stage.RPC,
+  timestamp = Instant.parse("2024-10-31T09:18:51Z"),
+  blockNumber = null,
+  transactionRLP = (
+    "0xb8d602f8d382e708018403780fc08403780fca830118fd8080b87960566023600b82828239805160001" +
+      "a607314601657fe5b30600052607381538281f3fe73000000000000000000000000000000000000000030" +
+      "146080604052600080fdfea2646970667358221220c399228be3777f235999f42c0d0a666dc163e66f600" +
+      "fdbfcc0a0500a4683db1064736f6c63430007060033c001a0a905ece7f3784afa2130063e332899fa60eb" +
+      "13863d96cea29810808c7d5a18eea0685b5237be1e44ccf7d4a9da4410a48cab5a23ba51e23fe3598294c7d34108c1"
+    ).decodeHex(),
+  reasonMessage = "Reverted transaction: 0x08c379a000000000000000000000000000000000000000000000000000000000000000" +
```

Comment:
Why are we adding this test case?

IIRC from our discussions, we agreed that this service **won't save reverted transactions**. If so, we should not have such a test which can be misleading of the intent of the service.

Reply:
Yes agree, to avoid confusion, I'd keep this rejected transaction instance but change the `reasonMessage` to be relevant to "traces-limit overflow", as I'd still like to have test cases for rejected contract deployment transactions which have no sender address in the decoded the RLP data